src/CommitmentsV2/SFA.DAS.CommitmentsV2/Models/AccountLegalEntity.cs

```diff
@@ -11,7 +11,7 @@ public class AccountLegalEntity : ICohortOriginator
         public string LegalEntityId { get; private set; }
         public string PublicHashedId { get; private set; }
         public Account Account { get; private set; }
-        public long AccountId { get; private set; }
+        public virtual long AccountId { get; private set; }
```

Comment:
Does this property need to be virtual? If it's for mocking is it possible to use the testing builders instead?